apollo-router-core/Cargo.toml

```diff
@@ -35,7 +35,7 @@ mockall = "0.10.2"
 static_assertions = "1"
 test-log = { version = "0.2.8", default-features = false, features = ["trace"] }
 tokio = { version = "1", features = ["full"] }
-tracing-subscriber = { version = "0.3.2", default-features = false, features = [
+tracing-subscriber = { version = "0.2.25", default-features = false, features = [
```

Comment:
Let's also have the comment `# don't bump to 0.3 until we bump tracing-opentelemetry to 0.16` here as well, it is super useful in the other location.

Reply:
done in c3aac0e